Pinta.Core/PaletteFormats/GimpPalette.cs

```diff
@@ -73,11 +79,21 @@ public void Save (IReadOnlyList<Color> colors, Gio.File file)
 		writer.WriteLine ("Name: Pinta Created {0}", DateTime.Now.ToString (DateTimeFormatInfo.InvariantInfo.RFC1123Pattern));
 		writer.WriteLine ("#");
 
-		foreach (Color color in colors) {
-			writer.WriteLine ("{0,3} {1,3} {2,3} Untitled", (int) (color.R * 255), (int) (color.G * 255), (int) (color.B * 255));
+		for (var i = 0; i < colors.Count; i++) {
```

Comment:
Can this revert back to being a `foreach` loop, or was there a reason for this change?

Reply:
@cameronwhite sorry for the confusion. Yes, indeed; the purpose was to give each color in the exported palette a unique name, but somehow I forgot to add that piece of code.

I just pushed two commits:
- The first one switches back to a `foreach` loop
- The second one switches (again) to a `for` loop, and gives each color a unique name based on the current index of the loop. If this is not desired, the commit can be discarded

Reply:
Thanks, that sounds fine